### PR TITLE
Add Parquet coverage to adapters CircleCI run

### PR DESCRIPTION
**Pull request by majetideepak**

### CODE REVIEW

**Comment by majetideepak** [comment]
I confirmed that Parquet tests are now covered
https://circleci.com/api/v1.1/project/github/facebookincubator/velox/16573/output/109/0?file=true&allocation-id=61bb7158b0f900312b0866b4-0-build%2F1D973EFC

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ workflows:
     jobs:
       - linux-build
       - linux-build-options
-      - linux-storage-adapters-build
+      - linux-adapters-build
```
Comment (mbasmanova) [inline .circleci/config.yml:21]: Looks good. I wonder if we could use a more specific name, e.g. linux-parquet-s3-build? adapters term is way too generic IMO.

Comment (majetideepak) [reply .circleci/config.yml]: made this change.

**Review by mbasmanova — APPROVED** [review]
Thank you, Deepak.

**Pull request closed without merging**